src/DonationForms/DataTransferObjects/DonationFormGoalData.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Give\DonationForms\DataTransferObjects;
 
+use Give\Campaigns\CampaignDonationQuery;
+use Give\Campaigns\Models\Campaign;
 use Give\DonationForms\DonationQuery;
 use Give\DonationForms\Models\DonationForm;
 use Give\DonationForms\Properties\FormSettings;
```

Comment:
Let's remove all unused imports from this file.

![image](https://github.com/user-attachments/assets/45effbb6-4f1e-493b-a86b-9df45fad60db)

Reply:
Resolved https://github.com/impress-org/givewp/pull/7569/commits/4ca37fe73d1c381bb8974334b0fec35f9d9386b0